### PR TITLE
Merging develop-userTypeFix in develop

### PR DESCRIPTION
**Pull request by AVtheking**
What kind of change does this PR introduce?

Feature

Issue Number:

Fixes https://github.com/PalisadoesFoundation/talawa-api/issues/1965

Did you add tests for your changes?

Yes

Summary

Merging the develop-userTypeFix branch in develop.

Does this PR introduce a breaking change?
Yes

Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
@Avtheking I'm going to close this PR. Use another source branch tests are going to fail

File: src/screens/OrgList/OrgList.tsx [hunk 1]
```diff
@@ -32,18 +32,22 @@ import useLocalStorage from 'utils/useLocalstorage';
 
 function orgList(): JSX.Element {
   const { t } = useTranslation('translation', { keyPrefix: 'orgList' });
-  const navigate = useNavigate();
   const [dialogModalisOpen, setdialogModalIsOpen] = useState(false);
   const [dialogRedirectOrgId, setDialogRedirectOrgId] = useState('<ORG_ID>');
-  function openDialogModal(redirectOrgId: string): void {
+
+  /* eslint-disable @typescript-eslint/explicit-function-return-type */
+  function openDialogModal(redirectOrgId: string) {
```
Comment (pranshugupta54) [inline src/screens/OrgList/OrgList.tsx:39]: Make the function `void` and remove eslint-disable

File: src/screens/OrgList/OrgList.tsx [hunk 2]
```diff
@@ -32,18 +32,22 @@ import useLocalStorage from 'utils/useLocalstorage';
...
     setdialogModalIsOpen(true);
   }
 
   const { getItem } = useLocalStorage();
+  const superAdmin = getItem('SuperAdmin');
+  const adminFor = getItem('AdminFor');
 
-  function closeDialogModal(): void {
+  /* eslint-disable @typescript-eslint/explicit-function-return-type */
+  function closeDialogModal() {
```
Comment (pranshugupta54) [inline src/screens/OrgList/OrgList.tsx:50]: Make the function `void` and remove eslint-disable

File: src/screens/OrgList/OrgList.tsx [hunk 3]
```diff
@@ -408,9 +410,8 @@ function orgList(): JSX.Element {
       {!isLoading &&
       ((orgsData?.organizationsConnection.length === 0 &&
         searchByName.length == 0) ||
-        (userData &&
-          userData.user.userType === 'ADMIN' &&
-          userData.user.adminFor.length === 0)) ? (
+        (userData && adminFor.length === 0)) ? (
+        // eslint-disable-next-line
```
Comment (pranshugupta54) [inline src/screens/OrgList/OrgList.tsx:414]: Remove eslint and wrap the complete div in `<> </>`

File: src/screens/Users/Users.tsx [hunk 4]
```diff
@@ -352,6 +359,7 @@ const Users = (): JSX.Element => {
           </h4>
         </div>
       ) : isLoading == false && usersData && displayedUsers.length === 0 ? (
+        // eslint-disable-next-line react/jsx-indent
```
Comment (pranshugupta54) [inline src/screens/Users/Users.tsx:362]: Remove disable and wrap complete in `<> </>`

**Review by pranshugupta54 — CHANGES_REQUESTED** [review]

**Comment by palisadoes** [comment]
@pranshugupta54 We are collaborating on this PR. You can ask @AVtheking to be a contributor to his repo and you can apply updates if you are interested.

**Comment by palisadoes** [comment]
@pranshugupta54 You can accept the invitation. It should be in your inbox

**Comment by palisadoes** [comment]
@varshith257 It's failing

- https://github.com/PalisadoesFoundation/talawa-admin/actions/runs/8397580202/job/23001165157?pr=1761
- It looks like it's only one file at fault.

image

**Comment by palisadoes** [comment]
@varshith257 Please fix the linting.

**Review by varshith257 — APPROVED** [review]
LGTM
Except low test coverage

**Comment by pranshugupta54** [comment]
For MemberDetail page, I already have the re-designed page with 100% coverage. We can merge that first, currently this file has very bad coverage so instead of fixing it for old design we can use the new one.
Ref: #1667

**Comment by varshith257** [comment]
@pranshugupta54 Okay, Thanks

**Comment by palisadoes** [comment]
@pranshugupta54 

1. We have to approve the GitHub action because you've never merged a PR before. This is going to be time consuming and cause delays.
2. Can you create a trivial PR in Admin that I can merge so that I don't have to approve subsequent workflows for you? For example add a space to a file

**Pull request merged**